### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by redwing555**
In this milestone of the rails blog app project, the following is done : 

- Creating forms to perform the following functions:

    1. Creates a Post on behalf of the `current_user`.
    2. Create a comment on behalf of the `current_user.`

- Allow Users to add likes to Posts.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
    root "users#index"
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+  resources :users, only: %i[index show] do
+    resources :posts, only: %i[index show new create]
+  end
+
+  resources :posts do
+    resources :comments, only: %i[create]
```
Comment (Dipeshtwis) [inline config/routes.rb]: - [ ] I would recommend adding `new`  in the comments resource. When we type `rails routes` it should be like below example code.

```ruby
 GET   /posts/:post_id/comments/new(.:format)     comments#new
 ```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @redwing555, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (redwing555) [reply config/routes.rb]: It is done, thank you @Dipeshtwis  for the review!

**Review by descholar-ceo — APPROVED** [review]
Hi @redwing555 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**